### PR TITLE
Task cms1

### PR DESCRIPTION
**Pull request by rrrichard**
admin page

### CODE REVIEW

File: admin_style.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    height: 100px;
+}
+
+.forms textarea{
+    float: right;
+}
+
+.forms input{
+    float: left;
+    /*margin-top: 50px;*/
```
Comment (aldofiore28) [inline admin_style.css]: Take this off

File: admin_style.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    font-size: 24px;
+    height: 100px;
+}
+
+.forms textarea{
+    float: right;
+}
+
+.forms input{
+    float: left;
```
Comment (aldofiore28) [inline admin_style.css]: This is not needed. The items are there already. If you don't want the space between them than you can leave it.

File: admin_page.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            <p>Edit paragraph</p>
+            <form>
+                <textarea rows="6" cols="80" name="edit_form"></textarea>
+                <h4>Choose which paragraph to edit</h4>
+                <select>
+                    <option>paragraph 1</option>
+                    <option>paragraph 2</option>
+                    <option>paragraph 3</option>
+                </select>
+                <div class="submit_buttons">
```
Comment (aldofiore28) [inline admin_page.php:39]: Could you not just give the inputs a margin top?

**Review by aldofiore28 — CHANGES_REQUESTED** [review]
Overall good, maybe you used too many divs, could have used more semantic tags.

File: admin_page.php [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                <textarea rows="6" cols="80" name="add_form"></textarea>
+                <input type="submit">
+            </form>
+        </div>
+        <div class="formSize">
+            <p>Edit paragraph</p>
+            <form>
+                <textarea rows="6" cols="80" name="edit_form"></textarea>
+                <h4>Choose which paragraph to edit</h4>
+                <select>
```
Comment (aldofiore28) [inline admin_page.php]: Needs a name to be referred to in the php.

File: admin_page.php [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                    <input type="submit" value="Edit Text">
+                    <input type="submit" value="Submit New Text">
+                </div>
+            </form>
+        </div>
+        <div class="formSize">
+            <p>Delete paragraph</p>
+            <form>
+                <h4>Choose which paragraph to delete</h4>
+                <select>
```
Comment (aldofiore28) [inline admin_page.php]: Needs a name to be referred to in the php.

File: admin_page.php [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+                    <option>paragraph 1</option>
+                    <option>paragraph 2</option>
+                    <option>paragraph 3</option>
+                </select>
+                <div class="submit_buttons">
+                    <input type="submit" value="Select Paragraph">
```
Comment (aldofiore28) [inline admin_page.php]: Needs a name to be referred to in the php.

File: admin_page.php [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+                    <input type="submit" value="Delete Paragraph">
```
Comment (aldofiore28) [inline admin_page.php]: Needs a name to be referred to in the php.

File: admin_page.php [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+            <form>
+                <textarea rows="6" cols="80" name="edit_form"></textarea>
+                <h4>Choose which paragraph to edit</h4>
+                <select>
+                    <option>paragraph 1</option>
+                    <option>paragraph 2</option>
+                    <option>paragraph 3</option>
+                </select>
+                <div class="submit_buttons">
+                    <input type="submit" value="Edit Text">
```
Comment (aldofiore28) [inline admin_page.php]: Needs a name to be referred to in the php.

File: admin_page.php [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+                    <input type="submit" value="Submit New Text">
```
Comment (aldofiore28) [inline admin_page.php]: Needs a name to be referred to in the php.

File: admin_page.php [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+            <h3>'ABOUT ME'</h3>
+        </div>
+    </div>
+
+    <div class="forms container">
+        <div class="formSize">
+            <p>Add additional paragraphs</p>
+            <form class="formAdd">
+                <textarea rows="6" cols="80" name="add_form"></textarea>
+                <input type="submit">
```
Comment (aldofiore28) [inline admin_page.php]: Needs a name to be referred to in the php.

**Review by aldofiore28 — CHANGES_REQUESTED** [review]

File: admin_style.css (hunk 1 above)
Comment (rrrichard) [reply admin_style.css]: yes sir

File: admin_style.css (hunk 2 above)
Comment (rrrichard) [reply admin_style.css]: thanks, it actually fixed my spacing problem

File: admin_page.php (hunk 3 above)
Comment (rrrichard) [reply admin_page.php:39]: it seems to mess things if I only target inputs

Comment (aldofiore28) [reply admin_page.php:39]: You can leave it then.

**Review by aldofiore28 — APPROVED** [review]
You can confirm the pull request now

**Pull request merged**